src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -1262,6 +1262,18 @@ export default class SourceFileLinter {
 					prop.initializer.text.startsWith("{") && prop.initializer.text.endsWith("}") &&
 					ts.isNewExpression(node) &&
 					this.#isPropertyBindingType(node, prop.name.getText())) {
+					const nodeSourceMap = this.sourceMaps?.get(this.resourcePath);
+					const traceMap = nodeSourceMap ? new TraceMap(nodeSourceMap) : undefined;
```

Comment:
Doing this check in here might be expensive, as the source map needs to be parsed over and over again.
What about instead storing the required information during xml transpiling in the `LintMetadata` and then just accessing it in here to decide whether a check should be performed or not?

Reply:
We need the traceMap for the original position of the node in the XML view.

Reply:
Yes, I understand that. But not in case we actually want to skip our check in JS, as the same check has already been performed during XML transpilation. So I wonder in which case we still need to report findings of XML sources in SourceFileLinter. If we do need it, we should only parse the sourceMap in that case, not to just check for the original file name. I suspect this would cause a lot of unnecessary overhead during linter runtime.

Reply:
Done